com.unity.formats.alembic/Editor/Importer/AlembicBuildPostProcess.cs

```diff
@@ -21,6 +22,12 @@ public static void OnPostProcessBuild(BuildTarget target, string pathToBuiltProj
         {
             if (!TargetIsSupported(target))
             {
+                if (FilesToCopy.Count > 0)
+                {
+                    Debug.LogError(
+                        "Alembic does not support your current standalone build target. The currently supported targets are: Win64, OSX and Linux 64. ");
```

Comment:
Suggestion:
"Alembic does not support your current standalone build target. You can currently choose only Windows 64-bit, MacOS X, or Linux 64-bit."
(+ remove extra space at the end of the string)

Reply:
how does this sound:
**Alembic only supports the following standalone build targets: Windows 64-bit, MacOS X, or Linux 64-bit.**

Shorter, and does not give hope that new targets will be added :)

Reply:
Right, in fact this was almost what I wrote for the doc :)
However, I checked in Unity's manual and it seems that `Standalone refers to builds for Mac, Windows and Linux.`
So I was wondering if we should remove "standalone" from here, because it could implicitly suggest that non-standalone targets are supported. Your opinion?